build.gradle

```diff
@@ -1,15 +1,14 @@
 plugins {
-    id 'java'
+    id 'java-library'
+    id 'jacoco'
     id 'checkstyle'
     id 'pmd'
-    id 'jacoco'
 }
 
 group 'no.unit.nva'
 version '1.0-SNAPSHOT'
 
-sourceCompatibility =  JavaVersion.VERSION_11  // source-code version and must be <= targetCompatibility
-targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
+
```

Comment:
since we are using java-11 features, I would recommend we keep the source and target compatibility

Reply:
They are moved into subprojects {}